Tweaks to ruby-ng.eclass to allow ruby-1.9 to work.

From: Daniel Robbins <drobbins@funtoo.org>


---
 profiles/base/use.mask |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/eclass/ruby-ng.eclass b/eclass/ruby-ng.eclass
index f1f324e..4b27580 100644
--- a/eclass/ruby-ng.eclass
+++ b/eclass/ruby-ng.eclass
@@ -268,6 +268,9 @@ _ruby_each_implementation() {
 			ree18)
 				_ruby_name=rubyee18
 				;;
+			ruby19)
+				_ruby_name=ruby1.9
+				;;
 		esac
 
 		RUBY=$(type -p $_ruby_name 2>/dev/null)
diff --git a/profiles/base/use.mask b/profiles/base/use.mask
index b9d21a3..f76a84a 100644
--- a/profiles/base/use.mask
+++ b/profiles/base/use.mask
@@ -246,9 +246,7 @@ dxr3
 pvm
 
 # Alex Legler <a3li@gentoo.org> (7 Dec 2009)
-# Ruby 1.9 is hardmasked
 # JRuby and REE18 are whitelisted on supported arches
-ruby_targets_ruby19
 ruby_targets_jruby
 ruby_targets_ree18
 
